src/component/project/Project/Project.styles.ts

```diff
@@ -36,13 +39,15 @@ export const useStyles = makeStyles()(theme => ({
         },
     },
     title: {
+        margin: 0,
+        width: '100%',
         fontSize: theme.fontSizes.mainHeader,
         fontWeight: 'bold',
         marginBottom: '0.5rem',
-        display: 'grid',
-        gridTemplateColumns: '1fr auto',
+        display: 'flex',
+        justifyContent: 'space-between',
         alignItems: 'center',
-        gridGap: '1rem',
+        gap: '1rem',
```

Comment:
Is this OK? Browser support is good, but not amazing.

Reply:
Hmm. How does it look in browser without support? Just ignores the gutters?

Reply:
Yes. Also, I don't think we should care for browsers from 2020 and back

![image](https://user-images.githubusercontent.com/2625371/169006348-10d3612a-9375-4fc5-8135-819494afb2d0.png)